java/core/src/java/org/apache/orc/impl/DictionaryUtils.java

```diff
@@ -46,6 +47,18 @@ public static void getTextInternal(Text result, int position,
     byteArray.setText(result, offset, length);
   }
 
+  public static ByteBuffer getTextInternal(int position, DynamicIntArray keyOffsets,
```

Comment:
Could you add a function description please, @belugabehr ?

Reply:
Done!